app/controllers/users_controller.rb

```diff
@@ -1,13 +1,15 @@
 class UsersController < ApplicationController
   before_action :authenticate_user!
 
-  # GET /users or /users.json
   def index
     @users = User.all
   end
 
-  # GET /users/1 or /users/1.json
   def show
+    @user = User.find(params[:id])
+    @follow = User.where(user_id: current_user.id)
```

Comment:
There is no difference

Reply:
so, `prams[:id]` is different from `current_user.id` I guess, if yes, then this should be ok, otherwise you are hitting the database for the same record twice